### DISCUSSION TITLE
Cloud password?

### DISCUSSION THREAD

**Discussion by DaveJS46** [osmandapp/OsmAnd#20050] [Q&A]
As a new user I've had a couple of tries with support to get an answer to the question I posed below:-
" _I started off using the app on my phone - free version.
I was so impressed I bought an 8" tablet to use for SatNav in the car and upgraded to a pro subscription.
I then opened a cloud account on the tablet and logged in using the emailed one time activation code. I thought that at this point it would be necessary to set a password to be able to log back into the cloud account if I had logged out of it, but as far as I can see there is no way to set up a password. If I log out of the cloud account (on the tablet) and try to log back in I just get another emailed one time activation code_. "

The replies, typically tell me how to set up an account, which I've already done, but not what I need to know, which a) Is a password necessay, b) if so how do I set it c) if not, do I always have to use a (new) emailed one time password every time I log in or d) should I never log out?

Any help would be appreciated.

**Comment by vshcherb** [comment]
Password is not needed cause it's not a web but a secured phone application, so you only need to pass first time registration.

**Reply by DaveJS46** [reply]
Thanks for your reply but every time I log out of Cloud I then have to log back in with the one time code that is emailed to me each time so in effect a new password for each log in, which seems a bit weird to me. Do I just never log out?

**Reply by vshcherb** [reply]
The token stored inside application is valid until you click logout, as we don't request to update it regularly or you don't use thin clients, it's anyway safe to send email every time new device is activated - so we don't bother to create a separate auth method. However on web you need to setup password and use it https://osmand.net/map

**Reply by DaveJS46** [reply]
OK, so basically what you are saying is "normally don't log out". But as and when I do then the "password" will be a new activation code. Still to my mind a bit weird, but I can live with it.
Probably it is just me - I don't normally use cloud services, maybe they are all run similarly.
